modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -8980,7 +8980,10 @@ private Scope _getScope(Group group) {
 				setExternalReferenceCode(group::getExternalReferenceCode);
 				setType(
 					() -> {
-						if (group.getType() == GroupConstants.TYPE_DEPOT) {
+						if (group.isCMS()) {
```

Comment:
Here you are using those methods (`isCMS`, `isDepot`, etc) instead of the value returned by the `getType()` method. I think we should also unify this [other code](https://github.com/liferay/liferay-portal/blob/6994b14cc04de46280b2a72ac4670d7ca563ffdd/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java#L792) (maybe there are more, I haven't checked it)

Reply:
I unified the code related to categories, but as discussed offline, the story is explicitly about categories and I'd thus exclude other types for now.